Fixed: Collapse button arrow points right when sidebar is collapsed

qKa opened the pull request:
- Adjusted the behavior of the Collapse button to ensure that its arrow icon points to the right when the sidebar navigation is collapsed.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-4)

qKa committed:
Fixed: Collapse button arrow points right when sidebar is collapsed

- Adjusted the behavior of the Collapse button to ensure that its arrow icon points to the right when the sidebar navigation is collapsed.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-4)
